docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -1,21 +1,98 @@
+When the `networkconfigs.net.liqo.io` exchanged by the two clusters are ready, the Liqo Network Manager creates a new resource of type `tunnelendpoints.net.liqo.io` which models the network interconnection between the two clusters.
+The `tunnelendpoints.net.liqo.io` is reconciled by the [Tunnel Operator](../gateway#tunnel-operator) and the [Route Operator](../route#route-operator).
 
 The following diagram illustrates how Liqo Network Manager creates the `tunnelendpoints.net.liqo.io`.
 
 ![Liqo Network Manager Simple Flow](../../../../images/liqonet/liqo-network-manager-simple-flow.png)
 
-#### Network Parameter Exchange
+
+#### Network Parameters Exchange
 
 The network parameters between two clusters are exchanged using the`networkconfigs.net.liqo.io` custom resources replicated between the clusters by the Liqo CRDReplicator component.
 
 ![Liqo Networkconfigs exchange flow](../../../../images/liqonet/networkconfigs-exchange-flow.png)
 
-1. The network configuration of Cluster1 is saved in a local custom resource (`NCFG 1->2`) containing the ClusterID of the remote cluster to whom is  destined. Same step done in Cluster2;
-2. `NCFG 1->2` is replicated to the remote cluster by the `CRDReplicator`, running in Cluster1 who knows how to interact with the API server of Cluster2. Same step done in Cluster2;
-3. In Cluster2 the custom resource `NCFG (1->2)` is processed and its status updated with the NAT information if any. The same is done with `NCFG (2->1)` in Cluster1.
-4. The CRDReplicator running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG (2->1)` it has all the necessary information to create the custom resource `TEP (1-2)` of type `tunnelendpoints.net.liqo.io`.
+1. The network configuration of Cluster1 is saved in a local custom resource (`NCFG(1->2)`) containing the ClusterID of the remote cluster to whom is destined. Same step done in Cluster2;
+2. The `CRDReplicator` running in Cluster1 (which knows how to interact with the API server of Cluster2) replicates `NCFG(1->2)` in the remote cluster. The same step is done in Cluster2;
+3. The custom resource `NCFG(1->2)` in Cluster2 is processed and its status updated with the NAT information (if any). The same is done with `NCFG(2->1)` in Cluster1.
+4. The `CRDReplicator` running in Cluster1 reflects this changes in the local resource `NCFG(1->2)`. Same step done in Cluster2; The TunnelEndpointCreator in Cluster1 combining the status of `NCFG(1->2)` and the spec of `NCFG(2->1)` it has all the necessary information to create the custom resource `TEP(1-2)` of type `tunnelendpoints.net.liqo.io`.
+
+
+### IPAM
+Embedded within the Liqo Network Manager, the IPAM (IP Address Management) is the Liqo module in charge of:
+1. Manage networks and IP addresses, as the name suggests.
```

Comment:
"Managing IP address" is obvious. Be more specific about what this component does.

Reply:
I have deleted this sentence. 
Since the IPAM currently manages only addresses belonging to the ExternalCIDR, I will describe this point later on, probably in the 'Endpoint IP translation with ExternalCIDR' Section